core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250211_add_workspace_spend_report_table.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="sehsan" id="create_WORKSPACE_SPEND_REPORT_TABLE">
+        <createTable tableName="WORKSPACE_SPEND_REPORT">
+            <column name="GOOGLE_PROJECT_ID" type="VARCHAR(254)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="REPORT_START_DATE" type="DATETIME">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="REPORT_END_DATE" type="DATETIME">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
+            <column name="TOTAL_SPEND" type="NUMBER(10,2)">
```

Comment:
If the data is unavailable, then shouldn't the spend columns be null, so shouldn't they be nullable?

Reply:
I thought we talked about making it zero during sprint planning? But maybe a `null` value would make the intention even clearer 🤔

Reply:
I think it's important to distinguish between actually $0.00 and "we just don't know".  maybe the data_available column would cover that, but having these null seems safer